core/src/main/resources/lib/form/repeatable/repeatable.js

```diff
@@ -173,19 +173,10 @@ Behaviour.specify("INPUT.repeatable-add", 'repeatable', 0, function(e) {
         e = null; // avoid memory leak
     });
 
-Behaviour.specify("INPUT.repeatable-delete", 'repeatable', 0, function(e) {
-        var b = makeButton(e,function(e) {
-            repeatableSupport.onDelete(e.target);
-        });
-        var be = $(b.get("element"));
-        be.on("mouseover",function() {
-            $(this).up(".repeated-chunk").addClassName("hover");
-        });
-        be.on("mouseout",function() {
-            $(this).up(".repeated-chunk").removeClassName("hover");
-        });
-
-        e = be = null; // avoid memory leak
+Behaviour.specify("BUTTON.repeatable-delete, INPUT.repeatable-delete", 'repeatable', 0, function(e) {
+        e.addEventListener("click", function() {
```

Comment:
Need to be careful here, I think make button adds some xss protection that this may be missing

Reply:
resolved in https://github.com/jenkinsci/jenkins/pull/5923/commits/59a680bf5ac3e5c4f9588be7827ea133359e60b3